guides/deployments/postgres.md

```diff
@@ -8,13 +8,18 @@ database.
 
 ## Background
 
-By default, Coder deploys a [TimescaleDB](https://www.timescale.com) inside the
-Kubernetes cluster to which you've installed Coder. However, we recommend this
-**only for evaluation purposes**, since the database isn't backed up and can be
-lost if the cluster goes down.
-
-As such, we strongly recommend using a PostgreSQL database for production
-deployments and hosting it **outside** the Kubernetes cluster hosting Coder.
+For convenience and ease of installation, Coder's default Helm Chart settings
+will  [PostgreSQL database](https://www.postgresql.org/) within the
+installation's Kubernetes namespace. This is useful for evaluation purposes;
+however, we **recommend using an out-of-cluster database for production**, to
+streamline maintenance operations, such as backups and upgrades. The database
+container is not backed up and will be lost when deleting the namespace or
```

Comment:
```suggestion
data is not backed up and will be lost when deleting the namespace or
```

Reply:
"database data" sounds funny to me so I went with "database state"